pkg/network/usm/tests/tracer_usm_linux_test.go

```diff
@@ -1401,6 +1407,12 @@ func testPostgresProtocolClassification(t *testing.T, tr *tracer.Tracer, clientH
 				serverAddress: serverAddress,
 				extras:        make(map[string]interface{}),
 			}
+			originalPostTracer := tt.postTracerSetup
+			wrapperPostTracer := func(t *testing.T, ctx testContext) {
+				cleanProtocolMapByProtocol(t, tr, protocols.Postgres)
+				originalPostTracer(t, ctx)
+			}
```

Comment:
Can we pass a `wrapperPostTracer` to the test itself, which will accept the protocol type and create it for each test?